pkg/parser/ast/stats.go

```diff
@@ -348,3 +349,61 @@ func (n *UnlockStatsStmt) Accept(v Visitor) (Node, bool) {
 	}
 	return v.Leave(n)
 }
+
+type RefreshStatsStmt struct {
+	stmtNode
+
+	RefreshObjects []*RefreshObject
+}
+
+func (n *RefreshStatsStmt) Restore(ctx *format.RestoreCtx) error {
+	ctx.WriteKeyWord("REFRESH STATS ")
+	for index, refreshObject := range n.RefreshObjects {
+		if index != 0 {
+			ctx.WritePlain(", ")
+		}
+		if err := refreshObject.Restore(ctx); err != nil {
+			return errors.Annotatef(err, "An error occurred while restore RefreshStatsStmt.RefreshObjects[%d]", index)
+		}
+	}
+	return nil
+}
+
+func (n *RefreshStatsStmt) Accept(v Visitor) (Node, bool) {
+	newNode, skipChildren := v.Enter(n)
+	if skipChildren {
+		return v.Leave(newNode)
+	}
+	n = newNode.(*RefreshStatsStmt)
+	return v.Leave(n)
+}
+
+type RefreshObjectScopeType int
+
+const (
+	RefreshObjectScopeTable RefreshObjectScopeType = iota + 1
+	RefreshObjectScopeDatabase
+	RefreshObjectScopeAll
+)
+
+type RefreshObject struct {
+	RefreshObjectScope RefreshObjectScopeType
+	DBName             string
```

Comment:
Should be `CIStr`, and maybe use `ast.TableName` so visitors can reuse existing logic.